### PR TITLE
BXMSDOC-4260: Added screenshots in the test scenario document.

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRAs:**
- [Epic](https://issues.redhat.com/browse/BXMSDOC-5719)
- [Dedicated JIRA](https://issues.redhat.com/browse/BXMSDOC-4260)

**Doc previews:**
- [RHPAM-Testing a decision service using test scenarios](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4260-RHPAM-TestScenario/#test-designer-create-test-template-rule-based-proc)
- [RHDM-Testing a decision service using test scenarios](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4260-RHDM-TestScenario/#test-designer-create-test-template-rule-based-proc)

**Doc impact:** Added images in the following sections.
- Section 4.1. Creating a test scenario template for rule-based test scenarios
- Section 7. Background instance in test scenarios

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-dmn-based-proc.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [id='test-scenarios-background-dmn-based-proc']
-= Adding a background data in DMN based test scenarios
+= Adding a background data in DMN-based test scenarios
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-dmn-based-proc.adoc:2]: Suggestion - rephrase references to "a background data":

Examples:
s/Adding a background data in/Adding background data objects to/Adding background data in
s/set a background data in/set background data objects in/set background data in

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc [hunk 2]
```diff
@@ -3,6 +3,17 @@
 
 In test scenario designer, you can use the *Background* tab to add and set the background data for rules-based as well as DMN based test scenarios. You can add and define the GIVEN data which is common for the entire test scenario simulation, based on the available data objects. *Background* tab has the ability to add and share the data among every test scenario. Data added using the *Background* tab can not be overridden by *Model* tab data.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc:4]: s/set the background data for/set background data for

Suggestion - simply sentence if possible and apply the hyphen to all instances of DMN based:

s/rules-based as well as DMN based/rules-based and DMN-based

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc [hunk 3]
```diff
@@ -8,7 +8,7 @@ After creating a test scenario template and defining the test scenarios, you can
 * To execute all the available test scenarios in your project inside multiple assets, in the upper-right corner of your project page, click *Test*.
 +
 .Run all the test scenarios from the project view
-image:AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
+image::AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
 +
 * To execute all the available test scenarios defined in a single `.scesim` file, at the top of the Test Scenario designer, click the *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc:13]: Minor suggestion - remove article and simplify:

s/To execute all the available test scenarios defined in a single `.scesim` file/To execute all available test scenarios defined in a `.scesim` file

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc:4]: Done

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc:13]: Done

**Pull request merged**